frontend/src/components/pages/camps/CamperDetailsBadges.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+import {
+  Container,
+  Text,
+  Image,
+  HStack,
+} from "@chakra-ui/react";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faHandDots,
+  faHandshakeAngle,
+} from "@fortawesome/free-solid-svg-icons";
+
+import icon_pickup from "../../../assets/icon_custom_sunrise.svg";
+
+import { Camper } from "../../../types/CamperTypes";
+
+export const CamperDetailsBadge = ({
+    customIcon,
+    icon,
+    description,
+    color,
+  }: {
+    customIcon: boolean;
```

Comment:
This works too, but another way to do this would be to make this field `customIconPath` an optional string. That way, when the string is present you use the local image and if there is no string, you can use the icon passed in into icon. The benefit of this would be that you have defined types and aren't using any. Up to you though, just a stylistic preference imo